.gitignore

```diff
@@ -26,12 +26,15 @@ src/modules.order
 /configs/*/emc.nml
 !/configs/common/emc.nml
 /docs/src/source-highlight/local
+docs/src/*/gui/gmoccapy_release_notes.txt
 lib/python/gremlin.py
 lib/python/qt5_graphics.py
 lib/python/touchy
 configs/sim/gmoccapy/*.pref
 configs/sim/gmoccapy/gmoccapy_plasma/*.pref
 configs/sim/gmoccapy/gmoccapy-JA/*.pref
+configs/**/*.tbl.bak
```

Comment:
Maybe even *.bak would make sense as a global rule?

Reply:
Perhaps, but that can wait for another patch.